activerecord/test/cases/fixtures_test.rb

```diff
@@ -73,8 +75,14 @@ def test_bulk_insert
       begin
         subscriber = InsertQuerySubscriber.new
         subscription = ActiveSupport::Notifications.subscribe("sql.active_record", subscriber)
-        create_fixtures("bulbs")
+        create_fixtures("bulbs", "authors", "computers")
+
         assert_equal 1, subscriber.events.size, "It takes one INSERT query to insert two fixtures"
+        if current_adapter?(:Mysql2Adapter)
```

Comment:
Beyond being ugly, we can't do this because other adapters use our test suite, and have their own name-quoting mechanisms.

Reply:
TIL, didn't know other adapters uses rails test suite. For my own curiosity could you point which one, I'm curious to see how they do to reuse the test suite